src/main/java/com/zufar/onlinestore/cart/converter/ShoppingSessionDtoConverter.java

```diff
@@ -1,14 +1,16 @@
 package com.zufar.onlinestore.cart.converter;
 
+import com.zufar.onlinestore.cart.api.ItemsTotalPriceCalculator;
 import com.zufar.onlinestore.cart.dto.ShoppingSessionDto;
 import com.zufar.onlinestore.cart.entity.ShoppingSession;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
 
-
-@Mapper(componentModel = "spring", uses = ShoppingSessionItemDtoConverter.class)
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING, uses = {ShoppingSessionItemDtoConverter.class, ItemsTotalPriceCalculator.class})
```

Comment:
maybe better use "spring" in the componentModel  instead of MappingConstants.ComponentModel.SPRING ?

Reply:
no